### PR TITLE
Fixup packageIndex & branding

### PR DESCRIPTION
**Pull request by wtgodbe**
Fixup the PackageIndex to include stable versions of `runtime.xyz-x64.runtime.native.System.IO.Ports`, and set the branding for 3.1 GA to `rtm`

CC @ericstj @safern @mmitche

### CODE REVIEW

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json [hunk 1]
```diff
@@ -185,16 +185,18 @@
         "2.2.0",
         "2.2.1",
         "2.2.2",
-        "3.0.0"
+        "3.0.0",
+        "3.1.0"
```
Comment (ericstj) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: This change is smaller than I expected.  I expected all packages to get a stable entry.  Perhaps you should run the target for AllConfigurations as well?

Comment (safern) [reply pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: Yeah. run the target with /p:BuildAllConfigurations=true.

Comment (ericstj) [reply pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: I went ahead and fixed this.

**Comment by wtgodbe** [comment]
Thanks @ericstj!

I verified that `build.cmd` locally updated the index as expected.

**Review by safern — APPROVED** [review]

**Comment by safern** [comment]
Just verified the artifacts of the build and they’re now stable! :smile:

**Pull request merged**